src/backend/distributed/operations/shard_split.c

```diff
@@ -673,7 +721,7 @@ CreateAndCopySplitShardsForShardGroup(HTAB *mapOfShardToPlacementCreatedByWorkfl
 								   workersForPlacementList);
 
 	DoSplitCopy(sourceShardNode, sourceColocatedShardIntervalList,
-				shardGroupSplitIntervalListList, workersForPlacementList);
+				shardGroupSplitIntervalListList, workersForPlacementList, NULL);
```

Comment:
Define a local variable with appropriate name and assign to NULL (the citus way)
or alternatively add a /* */ comment to describe param name.

Reply:
Done thanks.